Makefile

```diff
@@ -31,7 +31,7 @@ commit:
 
 .PHONY: update-deps
 update-deps:
-	glide update --no-recursive --strip-vcs
```

Comment:
Are you sure you want to do this? Why it does not work with `--no-recursive --strip-vcs`

Did you try to keep at least the `--strip-vcs` option?